File: src/com/mrpowergamerbr/loritta/commands/vanilla/discord/InviteCommand.java
package com.mrpowergamerbr.loritta.commands.vanilla.discord;

import com.mrpowergamerbr.loritta.commands.CommandBase;
import com.mrpowergamerbr.loritta.commands.CommandCategory;
import com.mrpowergamerbr.loritta.commands.CommandContext;

public class InviteCommand extends CommandBase {
	public String getDescription() {
		return "Envia o link do convite para adicionar a Loritta em outros servidores!";
	}

	public CommandCategory getCategory() {
		return CommandCategory.DISCORD;
	}

	@Override
	public String getLabel() {
		return "convite";
	}

	@Override
	public void run(CommandContext context) {
		context.sendMessage(context.getAsMention(true) + "https://discordapp.com/oauth2/authorize?client_id=297153970613387264&scope=bot&permissions=2080374975");
	}
}

File: src/main/java/com/mrpowergamerbr/loritta/commands/vanilla/discord/InviteCommand.kt
package com.mrpowergamerbr.loritta.commands.vanilla.discord

import com.mrpowergamerbr.loritta.Loritta
import com.mrpowergamerbr.loritta.commands.AbstractCommand
import net.perfectdreams.loritta.api.commands.CommandCategory
import com.mrpowergamerbr.loritta.commands.CommandContext
import com.mrpowergamerbr.loritta.utils.Constants
import com.mrpowergamerbr.loritta.utils.locale.LegacyBaseLocale
import net.dv8tion.jda.core.EmbedBuilder

class InviteCommand : AbstractCommand("invite", listOf("convidar", "convidarbot", "invitebot"), CommandCategory.DISCORD) {
    override fun getDescription(locale: LegacyBaseLocale): String {
        return locale["INVITE_DESCRIPTION"]
    }

    override suspend fun run(context: CommandContext,locale: LegacyBaseLocale) {
        var embed = EmbedBuilder()
                .setDescription(context.legacyLocale["INVITE_INFO", Loritta.config.addBotUrl, "${Loritta.config.websiteUrl}dashboard", "${Loritta.config.websiteUrl}support"])
                .setThumbnail("${Loritta.config.websiteUrl}assets/img/loritta_gabizinha_v1.png")
                .setColor(Constants.LORITTA_AQUA)
                .build()

        context.sendMessage(embed)
    }
}